standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/CompactionTxnHandler.java

```diff
@@ -875,7 +875,6 @@ public void cleanEmptyAbortedAndCommittedTxns() throws MetaException {
     try {
       Connection dbConn = null;
       Statement stmt = null;
-      ResultSet rs = null;
```

Comment:
please use `try-with-resources` instead of explicit resource management

Reply:
Hi @deniskuzZ thx for review. 
we are not just closing the db  connection in finally block, we are doing rollback in catch block(        rollbackDBConn(dbConn)).
I think try-with-resource may not work in this case. So i am keeping try block code only.

Reply:
try-with-resource works fine with that construct
````
try(Connection con = getConnection()) {
  try (PreparedStatement prep = con.prepareConnection("delete ...")) {
     ....
    con.commit()
  } catch (SQLException e) {
    ....
    con.rollback();
    ....
  }
}
````

Reply:
ok got it. done